crates/vm/levm/src/vm.rs

```diff
@@ -81,13 +94,25 @@ impl VM {
         calldata: Bytes,
         db: Arc<dyn Database>,
         mut cache: CacheDB,
+        access_list: AccessList,
     ) -> Result<Self, VMError> {
         // Maybe this decision should be made in an upper layer
 
         // Add sender, coinbase and recipient (in the case of a Call) to cache [https://www.evm.codes/about#access_list]
         let mut default_touched_accounts =
             HashSet::from_iter([env.origin, env.coinbase].iter().cloned());
 
+        let mut default_touched_storage_slots: HashMap<Address, HashSet<H256>> = HashMap::new();
+
+        for item in access_list.clone() {
```

Comment:
I would unpack this tuple
```suggestion
        for (address, keys) in access_list.clone() {
```
Something like this would be more clear.

Reply:
Done in [Change item for (address, keys) in both iterations](https://github.com/lambdaclass/ethrex/pull/1447/commits/142054a8ad143b3d2f3e02851614a82ee5a9accf)